platformio.ini

```diff
@@ -0,0 +1,25 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:esp32]
+platform = espressif32
+board = esp32-s3-devkitc-1
+framework = arduino
+build_flags = 
+	-DARDUINO_USB_MODE=1
+	-DARDUINO_USB_CDC_ON_BOOT=1
+	-I ./include
+	-I ./lib
+	-L ./lib
+lib_deps = 
+	adafruit/Adafruit NeoPixel
+	https://github.com/y-board/y-board-v3
+	adafruit/Adafruit GFX Library@^1.11.9
+	adafruit/Adafruit SSD1306@^2.5.10
```

Comment:
You don't need the NeoPixel library because y-board-v3 imports it for you. I'm pretty sure the same is true for the GFX library. Try removing these libraries and make sure your code still works.